lib/avo/menu/resource.rb

```diff
@@ -11,4 +12,8 @@ def parsed_resource
   def entity_label
     parsed_resource.navigation_label
   end
+
+  def fetch_params
+    params || {}
```

Comment:
Here we should consider that `params` can be a `proc`. We use [Avo::ExecutionContext](https://docs.avohq.io/3.0/execution-context.html) in this cases. Check how we use it [here](https://github.com/avo-hq/avo/blob/c7ca9b02b0734e1e23f22f95e152e78887b169e1/lib/avo/fields/base_field.rb#L189).

The `target` would be the `params` in this case and the other arguments are what we want to give access to the user inside the block.

Reply:
Done.

Reply:
Amazing, is the `resource` the instantiated on or should we pass `resource: parsed_resource` instead?